DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/queries/filteringResultsMultipleConditions/metadata.json

```diff
@@ -19,11 +19,12 @@
             "Type" : "link",
             "Url"  : "https://github.com/ravendb/book/blob/v4.0/Ch09/Ch09.md#querying-in-ravendb"
         },
-        {
-            "Title": "How to Query - todo",
-            "Type" : "link",
-            "Url"  : "https://ravendb.net/docs/article-page/4.2/java/client-api/session/querying/how-to-query"
-        },
+        //TODO
```

Comment:
does it compile? // it isn't valid json

Reply:
1. yes - it runs ok on my machine
2. issue opened: https://issues.hibernatingrhinos.com/issue/HRINT-1479